cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/workspace_overview.spec.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+const currentDate = Date.now();
+const workspaceName = 'foo-workspace' + currentDate;
+const workspaceDescription = 'foo-workspace-desc' + currentDate;
+
+if (Cypress.env('WORKSPACE_ENABLED')) {
+  let workspaceId = '';
+
+  describe('Workspace overview', () => {
+    before(() => {
+      cy.createWorkspace({
+        workspaceName,
+        description: workspaceDescription,
+        features: [
+          'management',
+          'discover',
+          'workspace_overview',
+          'workspace_update',
+          'dashboards',
+          'visualize',
+        ],
+      }).then((id) => {
+        workspaceId = id;
+      });
+      cy.log(`workspace ${workspaceName} create successfully`);
+    });
+
+    after(() => {
+      // cy.deleteWorkspaceById(workspaceId);
+    });
+
+    beforeEach(() => {
+      // Visit workspace overview page
+      cy.intercept('GET', `/w/${workspaceId}/api/workspaces/${workspaceId}`).as(
+        'getWorkspace'
+      );
+      miscUtils.visitPage(`/w/${workspaceId}/app/workspace_overview`);
+      cy.wait('@getWorkspace');
+    });
+
+    it('should successfully load the page', () => {
+      // workspace name is correctly displayed
+      cy.contains(workspaceName, { timeout: 60000 });
```

Comment:
Shall we need to add `.should('be.exists')`? `cy.contains` is not a assertion.

Reply:
you are right, `cy.contains` is not a assertion, it behaviors like `exist`, if element can't found, there will have error. 

I add `.should('be.visible')` for all contains here to make sure they are exist and visible.

Reply:
Yeah. It's a little bit confused. Some times the `contains` failed, but the whole test case passed. So I always should after contains.